.github/workflows/test.yml

```
@@ -0,0 +1,19 @@
+name: Test
+on: [push]
```

We probably want to follow the pattern we have in our other repos:

```suggestion
on:
  push:
    branches: ["main"]
  pull_request:

permissions:
  contents: read

```